### DISCUSSION TITLE
How to define custom presets?

### DISCUSSION THREAD

**Discussion by Atlas192** [yermak/AudioBookConverter#536] [Q&A]
am i stupid/blind, or is there no way to set a custom preset?
its a bit annoying to reset all the parameters after each start.
I know I can delete presets, but i think there is no way to add any.
Also I didn't see any file in the program folder to change the predefined presets.
I am running Version 6.2.10.

Apart form this issue,  the program works great for me.

**Comment by yermak** [comment]
@Atlas192 
Thanks for feedback, I need to think about usability and maybe there is some bug, which made it unclear...

However, let's start with simple thing - to change predefined preset - you need to change it. I mean it - select on of the predefined preset, change it (by selecting i.e. new bitrate or anything else) and it will be automatically saved and available for future use.

Adding preset - slighly more complicated - select one of the exising presets, in box with the name, start typing new name of the presset - hit Enter, when you completed typing - you should have new preset with settings from originally selected preset. Now you could change it according to you taste - see above.

P.S. I tried to make presets as simple and lightweigh as possible, probably went too far - people do expect menu, advanced settings, etc...
P.P.S. Added this information to hints.
